### ISSUE TITLE
A11y_AzureDataStudioAAE_PG1_Resource Health_DeveloperTool: No name property is provided for the combo box present under "Pods Present" header

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15757]
![15757_A11y_AzureDataStudioAAE_PG1_Resource Health_DeveloperTool](https://user-images.githubusercontent.com/85863543/127596444-bae1495e-2890-44e2-8c08-4059245b2575.png)
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#DevTools;#WCAG4.1.2;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "PG1" and open manage and tab to "Resource Health" page
5. Tab to the combo box present under "Pods Present" header and hit "ctrl + shift + i" and check if it has name provided.

### Actual:
No name property is provided for the combo box present under "Pods Present" header

### Expected:
Name property for the combo box present under "Pods Present" header should be assigned according to it's label which is "Select a pod in the dropdown below for detailed health information."

### User Impact:
Screen Reader users will not get the proper information about the combo box

**Comment by krrish2021** [comment]
@candiceye the issue can not be verified as the "Resource Health" control is not available in latest azure data studio @Charles-Gagnon Please reopen the bug.
![Screenshot (181)](https://user-images.githubusercontent.com/85863543/150959917-189a986d-bfc7-49f4-91bc-514b35ca769a.png)

**Comment by candiceye** [comment]
The combo box is already assigned according to its label, see this video:
[https://msit.microsoftstream.com/video/84b40840-98dc-a228-1d7c-f1ec8fbac03e](https://msit.microsoftstream.com/video/84b40840-98dc-a228-1d7c-f1ec8fbac03e)
